blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const marked = require('marked');
+
+var listOfDirectoryFiles = fs.readdirSync('markdown-blogs');
```

Comment:
the difference with var, const and let

* var - is the old variable type, its dynamic
* const - its a constant cant be resigned
* let - is basically a replacement for var, can be resigned

`listOfDirectoryFiles` can be a const